Makefile

```diff
@@ -61,13 +61,16 @@ dv_review_aks: ## make dv_review_aks deploy APP_NAME=2222 CLUSTER=cluster1 USE_D
 
 qa_aks:
 	$(eval include global_config/qa_aks.sh)
+	$(eval DEPLOY_ENV=qa)
```

Comment:
DEPLOY_ENV is already defined in the global_config eg: https://github.com/DFE-Digital/publish-teacher-training/blob/main/global_config/qa_aks.sh#L2
Let's not redefine it as it's confusing, but make it work with the value from global_config (ie rename your files)